packages/spark-core/src/spark-core.js

```diff
@@ -210,12 +210,6 @@ const SparkCore = AmpState.extend({
   logout(...args) {
     return Promise.resolve()
       .then(() => {
-        if (this.mercury) {
-          return this.mercury.disconnect();
-        }
-        return Promise.resolve();
-      })
-      .then(() => {
```

Comment:
The mercury disconnect is not needed since as soon as we disconnect, the web-client starts synchronizing due to a mercury state change and results in data fetches. As a result it slows down the logout process. Mercury would anyways be disconnected as soon as the logout is done and the client is refreshed.